master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -64,7 +66,7 @@ public abstract class AbstractMetaManager implements IMetadataHandler {
   public final Map<String, Set<Integer>> registeredAppAndShuffles =
       JavaUtils.newConcurrentHashMap();
   public final Set<String> hostnameSet = ConcurrentHashMap.newKeySet();
-  public final Set<WorkerInfo> workers = ConcurrentHashMap.newKeySet();
+  private final Map<String, WorkerInfo> workersMap = JavaUtils.newConcurrentHashMap();
```

Comment:
Keep this field public can remove these methods below. Although there might be some incorrect usable of this map, I believe that the reviews can find out.

Reply:
sure

Reply:
addressed. thanks